app/uk/gov/hmrc/tai/viewModels/income/ConfirmAmountEnteredViewModel.scala

```diff
@@ -52,6 +52,23 @@ object ConfirmAmountEnteredViewModel {
     )
   }
 
+  def annualPayCurrentYear(employmentId: Int, employerName: String, currentAmount: Int, estimatedIncome: Int)(implicit messages: Messages): ConfirmAmountEnteredViewModel = {
```

Comment:
I think there is an opportunity to clean up this object as there is quite a bit of repeating code. I think at the very least the vals could be in-lined with common ones pushed up. Other options would be to implement a factory or something that resembles "a template method in a super class"

Reply:
looks like this is done now?

Reply:
Yes I managed to take a look, It just needs a review now.